proc_echo_mw.py

```diff
@@ -22,85 +23,41 @@
     s = find_file(searchstr, exp_type=exp_type, expno=nodename,
             postproc=postproc,
             lookup=postproc_dict)
-    
-    #{{{Fourier Transforms coherence channels
-    s.ft(['ph1','ph2']) 
-    #}}}
-
-    #{{{visualize the frequency limits
+    s.ft(['ph1','ph2']) # Fourier Transforms coherence channels
     fl.next('all data: frequency domain')
     fl.image(s)
     fl.side_by_side('show frequency limits\n$\\rightarrow$ use to adjust freq range',
-            s,freq_range)
-    #}}}
-
-    #{{{slice out the frequency range along t2 axis
-    s = s['t2':freq_range]
-    #}}}
-    
-    #{{{inverse fourier transform into time domain
-    s.ift('t2')
-    #}}}
-    
-    #{{{visualize time domain after filtering and phasing 
-    logger = init_logging("info")
-    logger.info(strm("THIS IS THE SHAPE"))
-    logger.info(strm(ndshape(s)))
+            s,freq_range) # visualize the frequency limits
+    s = s['t2':freq_range] # slice out the frequency range along t2 axis
+    s.ift('t2') # inverse fourier transform into time domain
+    logger.debug(strm("THIS IS THE SHAPE"))
+    logger.debug(strm(ndshape(s)))
```

Comment:
most of this is just formatting (doesn't change what the code does) I do change the logger statement from info to debug, b/c it's a debugging type of statement